client/src/components/app.js

```diff
@@ -51,6 +57,15 @@ class App extends React.Component {
     });
   }
 
+  renderMenuActions() {
+    const { logoutUser } = this.props;
+
+    return [
```

Comment:
Note: Moving the masthead pieces into here was a good start. Not sure how feel about all the authentication bits spread out, maybe at some point a single function/filter inside of ```App``` to handle it would be nice.

Note: Missed this before, but tracking the ```props.wasLoggedIn``` value was interesting. It's only mentioned in one place, and not in the initial userState. Might be a good idea for consistency to place it in the initial userState.